packages/dapp/src/hooks/useClaim.js

```diff
@@ -139,15 +144,19 @@ export const useClaim = () => {
         );
       }
       let message =
-        txMessage &&
-        txMessage.signatures &&
-        txMessage.signatures.length >= homeRequiredSignatures
+        txMessage && txMessage.messageData && txMessage.signatures
           ? txMessage
           : null;
       if (!message) {
         const homeProvider = await getEthersProvider(homeChainId);
         message = await getMessage(true, homeProvider, homeAmbAddress, txHash);
       }
+      message.signatures = getRemainingSignatures(
```

Comment:
Here we no longer check the number of collected signatures. `getRemainingSignatures` might return less signatures then needed in case they are not present in files for some reason. In this case, afaik, 
`TOKENS_CLAIMED` error will be returned from `executeCallback(message)`.

We should return `NOT_ENOUGH_COLLECTED_SIGNATURES` instead, if `message.signatures.length < requiredSignatures`

Reply:
Thank you. I'm adding a check to throw if there are not enough signatures